ReleaseNotes.md

```diff
@@ -1,5 +1,53 @@
+
+- Features:
+  - (OpenVINO) Added modified AWQ algorithm for 4-bit data-aware weights compression. This algorithm applied only for patterns `MatMul->Multiply->Matmul`. For that `awq` optional parameter has been added to `nncf.compress_weights()` and can be used to minimize accuracy degradation of compressed models (note that this option increases the compression time).
+  - (ONNX) Introduced support for the ONNX backend in the `nncf.quantize_with_accuracy_control()` method. Users can now perform quantization with accuracy control for `onnx.ModelProto`. By leveraging this feature, users can enhance the accuracy of quantized models while minimizing performance impact.
+  - (ONNX) Added an example based on the YOLOv8n-seg model for demonstrating the usage of quantization with accuracy control for the ONNX backend.
+  - (PT) Added SmoothQuant algorithm for PyTorch backend in `nncf.quantize()`.
+  - (OpenVINO) Added an example with the hyperparameters tuning for the TinyLLama model.
+  - Introduced the `nncf.AdvancedAccuracyRestorerParameters`.
+  - Introduced the `subset_size` option for the `nncf.compress_weights()`.
+  - Introduced `TargetDevice.NPU` as the replacement for `TargetDevice.VPU`.
+- Fixes:
+  - Fixed API Enums serialization/deserialization issue.
+  - Fixed issue with required arguments for `revert_operations_to_floating_point_precision` method.
+- Improvements:
+  - (ONNX) Aligned statistics collection with OpenVINO and PyTorch backends.
+  - Extended `nncf.compress_weights()` with Convolution & Embeddings compression in order to reduce memory footprint.
+- Deprecations/Removals:
+  - (OpenVINO) Removed outdated examples with `nncf.quantize()` for BERT and YOLOv5 models.
+  - (OpenVINO) Removed outdated example with `nncf.quantize_with_accuracy_control()` for SSD MobileNetV1 FPN model.
+  - (PyTorch) Deprecated the `binarization` algorithm.
+  - Removed Post-training Optimization Tool as OpenVINO backend.
+  - Removed Dockerfiles.
+  - Removed the `TargetDevice.VPU` device.
+- Tutorials:
+  - [Post-Training Optimization of Stable Diffusion v2 Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/236-stable-diffusion-v2/236-stable-diffusion-v2-text-to-image.ipynb)
+  - [Post-Training Optimization of DeciDiffusion Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/259-decidiffusion-image-generation/259-decidiffusion-image-generation.ipynb)
+  - [Post-Training Optimization of DepthAnything Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/280-depth-anything/280-depth-anything.ipynb)
+  - [Post-Training Optimization of Stable Diffusion ControlNet Model](https://github.com/openvinotoolkit/openvino_notebooks/tree/main/notebooks/235-controlnet-stable-diffusion/235-controlnet-stable-diffusion.ipynb)
+
+Compression-aware training:
+
+- Fixes
+  - (PyTorch) Fixed issue with `NNCFNetworkInterface.get_clean_shallow_copy` missed arguments.
+
```

Comment:
Are you going to add "Thanks to our contributors" in this release https://github.com/openvinotoolkit/nncf/issues/2499?

Reply:
Do we need it in `ReleaseNotes.md`? Which format do you propose to use in the document?

Reply:
I think the release notes is good place to say thanks to our contributors. Format is following:
### Thanks to our Contributors
contributor_name_0, contributor_name_1 and etc.

Reply:
I'll place the information about contributors in the GitHub notes during the release. Do we need it in the `ReleaseNotes.md` **document**?

Reply:
Good suggestion! It's enough.